### PR TITLE
SDIT-2237: ✨ Uppercase and trim string fields

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/historicalprisonerapi/integration/IndentifierSearchResourceIntTest.kt [hunk 1]
```diff
@@ -116,5 +130,33 @@ class IndentifierSearchResourceIntTest : IntegrationTestBase() {
         .expectStatus()
         .isBadRequest
     }
+
+    @Test
+    fun `should ignore blank other identifiers`() {
+      webTestClient.get()
+        .uri("/identifiers?prisonNumber=AB111111&pnc= &cro= ")
```
Comment (karenmillermoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/historicalprisonerapi/integration/IndentifierSearchResourceIntTest.kt:137]: Can we also have a test where the entry is totally empty e.g. pnc=&

**Review by karenmillermoj — APPROVED** [review]
lgtm
